js/ui/popup/popup_drag.js

```diff
@@ -131,7 +131,7 @@ class PopupDrag {
     }
 
     _getContainerDimensions() {
-        const document = domAdapter.getDocument();
+        const document = domAdapter.getRootNode(this._draggableElement);
```

Comment:
Why do we need this change? Will this code work `rootNode.body.clientHeight`?

Reply:
I removed this change